api/src/queries/GetControlActionByTargetQuery.js

```diff
@@ -0,0 +1,44 @@
+import StringHelper from '../helpers/StringHelper'
```

Comment:
My code breaks on this spread operator: (Node 10.13)

/Users/wim/.nvm/versions/node/v10.13.0/bin/node -r babel-register /Users/wim/TROMPA/CE-api/api/src/index.js
/Users/wim/TROMPA/CE-api/api/node_modules/babel-core/lib/transformation/file/index.js:558
      throw err;
      ^

SyntaxError: /Users/wim/TROMPA/CE-api/api/src/queries/GetControlActionByTargetQuery.js: Unexpected token (23:34)
  21 | 
  22 |     // split the params into targetParams and baseParams
> 23 |     const { target: targetParams, ...baseParams } = this.params

Reply:
Fixed by merging branch `feat/update-babel`